src/loaders/AudioLoader.js

```diff
@@ -33,6 +33,10 @@ AudioLoader.prototype = Object.assign( Object.create( Loader.prototype ), {
 
 					onLoad( audioBuffer );
 
+				}, function ( error ) {
+
+					throw error;
```

Comment:
How about using `onError( e );` instead?

Reply:
That would skip the LoadingManager's error handler. Does it matter?

Reply:
If you throw an exception at this point, it is not handled by the `catch` block in line 42. Ideally, errors in the loading process always trigger the `onError()` callback so the user can decide how to proceed. The loading manager could be informed about the error like so:
```js
if ( onError ) {

	onError( e );

} else {

	console.error( e );

}

scope.manager.itemError( url );
```

Reply:
Although I'm not sure `LoadingManager` can process the call of `itemError()` correctly since it would be trigger during the audio decode. At that point, `itemEnd()` was already called by `FileLoader`. 

The fact that `AudioLoader` does a decode can actually produce a race condition: #10706 (not sure why the issue was closed since this bug still exists).